### ISSUE TITLE
some questions about DAN_Model

### ISSUE THREAD

**Issue by JWSunny** [MarekKowalski/DeepAlignmentNetwork#31]
hi :
1. i  train the first stage only and then the second stage only；but when i trained the first stage, the validation set was chosen at random from train data sets, epoch=200, lr=0.001,  is this process suitable?

looking forward to your reply,  thanks.

**Comment by MarekKowalski** [comment]
Hi,

I usually do not train for a specific number of epochs, like you specified here but rather till the valdation error stops decreasing rapidly. As for the rest: it looks ok!

Marek
